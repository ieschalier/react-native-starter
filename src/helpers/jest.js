/* eslint-disable import/prefer-default-export, import/no-extraneous-dependencies */
import configureStore from 'redux-mock-store'

export const createMockReduxStore = (initialState) => {
  const mockStore = configureStore()

  return mockStore(initialState)
}
