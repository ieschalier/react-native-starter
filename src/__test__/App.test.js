import React from 'react'
import renderer from 'react-test-renderer'
import { createMockReduxStore } from '../helpers/jest'
import App from '../index'

it('renders without crashing', () => {
  const store = createMockReduxStore({})
  const rendered = renderer.create(<App store={store} />)
  expect(rendered.toJSON()).toMatchSnapshot()
})
