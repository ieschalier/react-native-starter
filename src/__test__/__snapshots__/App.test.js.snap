// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "#fff",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "center",
      },
      undefined,
    ]
  }
/>
`;
